php/src/vaas/Vaas.php

```diff
@@ -203,15 +183,8 @@ public function forStreamAsync(ReadableStream $stream, int $fileSize, ?ForStream
                     throw new VaasClientException('Stream is not readable');
                 }
 
-                if ($options === null) {
-                    $options = new ForStreamOptions(
-                        [
-                            'vaasRequestId' => null,
-                            'timeout' => $this->options->timeout,
-                            'useHashLookup' => $this->options->useHashLookup ?? true,
-                        ]
-                    );
-                }
+                $options = $options ?? ForStreamOptions::default();
```

Comment:
Same here as before:
```suggestion
               if ($options === null) {
                    $options = new ForStreamOptions($this->options->useHashLookup, 300, null);
                }
```